CODEOWNERS

```diff
@@ -64,6 +64,7 @@ Kconfig*                                  @tejlmand
 /drivers/serial/                          @nordic-krch @anangl
 /drivers/sensor/paw3212/                  @anangl @pdunaj @MarekPieta
 /drivers/sensor/pmw3360/                  @anangl @pdunaj @MarekPieta
+/drivers/wifi/nrf700x/                    @krish2718 @sachinthegreen @sr1dh48r
```

Comment:
You may consider adding someone from Vestavind as well for swifter merges (can be me).